### PR TITLE
Error handling common

### PR DESCRIPTION
**Pull request by sashrika**
Error handling in few resources.

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPersonResourceProvider.java [hunk 1]
```diff
@@ -127,9 +127,9 @@ public MethodOutcome createFHIRPerson(@ResourceParam Person person) {
 	public MethodOutcome updatePersonConditional(@ResourceParam Person thePerson, @IdParam IdDt theId) {
 		MethodOutcome retVal = new MethodOutcome();
 		OperationOutcome outcome = new OperationOutcome();
-		try {
-			Person person = personResource.updateFHIRPerson(thePerson, theId.getIdPart());
-		} catch (Exception e) {
+		//try {
```
Comment (surangak) [inline omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPersonResourceProvider.java]: Why the commented out try catch?

File: api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandler.java [hunk 2]
```diff
@@ -44,42 +44,42 @@
 import java.util.Set;
```
Comment (surangak) [inline api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandler.java]: You didn't really change this file, did you? :-)

Comment (sashrika) [inline api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandler.java]: @harsha89 , not changed, formatted it :) .

File: omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPersonResourceProvider.java (hunk 1 above)
Comment (sashrika) [inline omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPersonResourceProvider.java]: @surangak  , I removed this generic exception. Now the PersonServiceImpl.java file will validate the request body in fine grained level and give a more meaningful message.

**Comment by harsha89** [comment]
pull request can't merge can you fix conflicts?

**Pull request closed without merging**